editor/src/components/canvas/controls/text-edit-mode/text-edit-mode.spec.browser2.tsx

```diff
@@ -64,14 +82,22 @@ describe('Click to choose target text for editing', () => {
     const editor = await renderTestEditorWithCode(projectWithNestedDiv, 'await-first-dom-report')
 
     pressKey('t')
-    await selectElement(editor, 'div')
+    await clickOnElement(editor, 'div')
```

Comment:
does this imply that eg `<button>Hello World</button>`  wouldn't be huntable / editable with `t` ?

Reply:
That shouldn't make a difference since this is in select mode and not live mode, so the button itself wouldn't be handling the events